files/common/var/lib/delphix-platform/export-home

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Copyright (c) 2025 by Delphix. All rights reserved.
+#
+
+#
+# This script ensures that the /export/home is a symlink
+# to /home.
+#
+
+if ! mountpoint -q /export/home; then
+  if [ ! -L /export/home ]; then
+    echo 'Ensuring /export/home is a symlink to /home...'
+    if [ -e /export/home ]; then
+      echo 'Removing existing /export/home directory...'
+      rm -rf /export/home
```

Comment:
This seems risky. If there's some bug in our code and we fail to unmount home directories for whatever reason, this will delete all home directories. At this point, if everything works as designed, `/export/home` should be an empty directory (it's the mountpoint for a filesystem that should have been unmounted already), and so `rmdir` should succeed.